internal/controllers/submariner/loadbalancer_resources.go

```diff
@@ -108,14 +110,18 @@ func newLoadBalancerService(instance *v1alpha1.Submariner, platformTypeOCP strin
 		svcAnnotations = map[string]string{}
 	}
 
+	if instance.Spec.HostedCluster {
```

Comment:
Thanks for adding the flag to Submariner CRD.
Overall code LGTM , 
Only wondering 
A. If we should apply this change only for hosted KubeVirt at first until we verify it also for other hosted platforms (e.g hosted ROSA) 
B. if this new field might affect upgrade ?

Reply:
A) Chances of working with external traffic policy cluster is higher in that case right ? As they might have similar implementations. We can add a separate check if needed for any platforms.
B) It is an optional field and it should take the default value false in an upgraded cluster.

Reply:
A. I successfully deployed Submariner on ROSA HCP a few months ago (with current LB code), so not sure how setting up an external traffic policy cluster will affect ROSA HCP.  I think we should consider making this change only for hosted Kubevirt at this point

B. I meant upgrading the HCP cluster use case, anyway I think we can address that once we know what logic we are adding in the hosted case